modules/fluid_properties/doc/content/source/materials/SodiumPropertiesMaterial.md

```diff
@@ -0,0 +1,28 @@
+# Sodium Properties Material
+
+!syntax description /Materials/SodiumPropertiesMaterial
+
+`SodiumPropertiesMaterial` samples the functions availble in [SodiumProperties](SodiumProperties.md)
+and provides them as material properties.
+
+The following materials created for liquid sodium are:
+
+- Thermal Conductivity: `k(temperature)`
+- Enthalpy: `h(temperature)`
+- Heat capacity: `heatCapacity(temperature)`
```

Comment:
If this refers to the actual material property name, then it should be `cp`, `heatCapacity` is the name of the method.